src/QMCWaveFunctions/ElectronGas/FreeOrbitalBuilder.cpp

```diff
@@ -13,10 +13,12 @@ FreeOrbitalBuilder::FreeOrbitalBuilder(ParticleSet& els, Communicate* comm, xmlN
 std::unique_ptr<SPOSet> FreeOrbitalBuilder::createSPOSetFromXML(xmlNodePtr cur)
 {
   int npw, norb;
+  std::string spo_object_name;
   PosType twist(0.0);
   OhmmsAttributeSet attrib;
   attrib.add(norb, "size");
   attrib.add(twist, "twist");
+  attrib.add(spo_object_name, "name");
```

Comment:
If id is intended to be an alternate input tag for the spo_object_name it is missing here.

Reply:
In a couple of places, `name` and `id` are interchangeable. In this PR, I added both because there are existing input files. It is unclear how we would like to move forward. My current thought is to disable "id" support and allow "name" only. Supporting both requires do a lot of extra things, for example trapping errors when "name" and "id" differ.

For now, we need both both. Here is what I will do. 
```
attrib.add(spo_object_name, "name");
attrib.add(spo_object_name, "id", {}, TagStatus::DEPRECATED)
```
and later change to TagStatus::DELETED.

I will do a separate to address this change.